api/services/get.js

```diff
@@ -5,18 +5,18 @@ module.exports = self;
 
 var async = require('async');
 var _ = require('underscore');
-var fs = require('fs');
-var path = require('path');
-
 var configHandler = require('../../common/configHandler.js');
+var path = require('path');
 
 function get(req, res) {
   var bag = {
     reqQuery: req.query,
     resBody: [],
     initializeDefault: false,
     component: 'services',
-    defaultService: {}
+    defaultService: {},
+    services: require(path.join(global.config.scriptsDir,
```

Comment:
not sure why are we doing this. the previous workflow is doing better error handling and exits gracefully. is there anything extra we're getting from loading script like this?